packages/workers/src/scripts/il-alerts-discord.ts

```diff
@@ -0,0 +1,139 @@
+import dotenv from 'dotenv';
```

Comment:
Can we re-use some of this code between `il-alerts` and `il-alerts-discord`? Basically everything except the logic to actually post is the same, right?

Reply:
I was going to just use il-alerts but the messages have to be different between the 2 because discord doesn't like html. Also on the discord version I was told to have only the positive alerts.  With those differences and in process to get this out the door quickly I hacked this.

Reply:
Alright, fair enough - you owe me a refactor if and when we add a third communication channel 😄